submissions/promiseChilufya/Database/init.sql

```diff
@@ -0,0 +1,50 @@
+CREATE database food_bag;
+
+USE food_bag;
+/* This is the food table */
+
+	CREATE TABLE food_table(
+	id INT NOT NULL AUTO_INCREMENT,
+	food_type VARCHAR(255) NOT NULL,
```

Comment:
The food type is of four different food categories. This is better as ENUM (1, 2, 3, 4)